src/main/java/core/basesyntax/service/impl/operation/BalanceOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl.operation;
+
+import core.basesyntax.service.OperationHandler;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public int executeOperation(int initBalance, int quantity) {
```

Comment:
The method `executeOperation` should have a more descriptive name that reflects its purpose of handling the 'balance' operation.

Reply:
++